lms/djangoapps/learner_home/test_serializers.py

```diff
@@ -820,7 +820,7 @@ def create_test_data(cls, enrollment):
         """Mock data following the shape of credit_statuses"""
 
         return {
-            "course_key": enrollment.course.id,
+            "course_key": str(enrollment.course.id),
```

Comment:
Note that the return from the API *also* includes this string of the course key, but inside the object value instead of the key.